src/server/renderToString.ts

```diff
@@ -7,6 +7,7 @@ import {
 	copyPropsTo
 } from '../DOM/utils';
 import {
+	EMPTY_OBJ,
```

Comment:
May sound strange but rather than import like this, it's better to do:
`import { EMPTY_OBJ } from 'inferno'`